subprojects/docs/src/docs/userguide/introduction_dependency_management.adoc

```diff
@@ -73,3 +73,10 @@ Gradle takes your dependency declarations and repository definitions and attempt
 * All of the artifacts for the module are then requested from the _same repository_ that was chosen in the process above.
 
 The dependency resolution process is highly customizable to meet enterprise requirements. For more information, see the chapter on <<customizing_dependency_resolution_behavior.adoc#customizing_dependency_resolution_behavior,customizing dependency resolution>>.
+
+[[sub:http-retries]]
+### HTTP Retries
+
+Gradle will make several attempts to connect to a given repository. If the first HTTP call fails, Gradle will retry increasing the amount of time between each retry. After the max number of failed attempts, the repository will be blacklisted.
+
+The number of attempts and length of time between attempts is not configurable.
```

Comment:
:x: Let's remove this, this is actually not true, there _is_ a way to do it, we just don't want it to be public at this stage.

Reply:
^^ @eriwen